dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/transform/ContentletTransformerTest.java

```diff
@@ -166,6 +175,68 @@ public String getUrl(Contentlet contentlet) {
         assertEquals(urlExpected, newContentlet.getMap().get(HTMLPageAssetAPI.URL_FIELD));
         assertFalse(newContentlet.getMap().containsKey(Contentlet.NULL_PROPERTIES));
     }
+    
+    /**
+     * this test creates a widget type that has constant fields and a content of this type.
+     * It updates the constant field values and insures that the new constant field values are both
+     * in the content object and the newly transformed map
+     * @throws Exception
+     */
+    @Test
+    public void Test_Constant_fields_are_in_map() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4843681)